custom_components/weenect/device_tracker.py

```diff
@@ -179,5 +179,14 @@ def extra_state_attributes(self):
                 res["speed"] = self.coordinator.data[self.id]["position"][0]["speed"]
                 res["course"] = self.coordinator.data[self.id]["position"][0]["direction"]
                 res["PDOP"] = self.coordinator.data[self.id]["position"][0]["pdop"]
+                res["last_position_date"] = self.coordinator.data[self.id]["position"][0]["date_tracker"]
+                res["gsm_text"] = self.coordinator.data[self.id]["position"][0]["gsm_text"]
+                res["gsm_color"] = self.coordinator.data[self.id]["position"][0]["gsm_color"]
+                res["accuracy_text"] = self.coordinator.data[self.id]["position"][0]["accuracy_text"]
+                res["accuracy_color"] = self.coordinator.data[self.id]["position"][0]["accuracy_color"]
+                res["battery_text"] = self.coordinator.data[self.id]["position"][0]["battery_text"]
+                res["battery_color"] = self.coordinator.data[self.id]["position"][0]["battery_color"]
```

Comment:
What is the added value in providing these in addition to `battery`?

Reply:
same as accuracy, it's the «official hint» available in the official app